english/matrix/matrix.md

```diff
@@ -22,3 +22,6 @@ belly and see the outside world.
 It's dark and fluffy. 
 While you don't believe that you might be the One,
  it's only up to you to save the world.
+
+[You crawl through the hole you just made and...](english/matrix/matrix2.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.